### PR TITLE
Add section about cryptography in the README

### PR DESCRIPTION
**Pull request by denji**

### CODE REVIEW

File: crypto_guidelines.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ in depth rather than due to any particular known exploit techniques.
 ### 1) Do not conditionally choose between constant and non-constant time
 
 It may be natural in crypto code to do the equivalent of:
-```
+```cpp
```
Comment (chandlerc) [inline crypto_guidelines.md]: My reading is that these should be `c` or `c++`? But I'll admit I'm not a github / markdown expert.

File: crypto_guidelines.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ If your cryptographic code tries to scrub private key data from registers or
 local buffers used during a constant-time algorithm, that scrubbing needs to be
 unconditional to be effective. For example, don't do things like this (somewhat
 silly) example:
-```
+```cpp
```
Comment (chandlerc) [inline crypto_guidelines.md]: Same comment.

**Review by chandlerc — CHANGES_REQUESTED** [review]
Thanks for the cleanups! Two nits below.

File: crypto_guidelines.md (hunk 1 above)
Comment (denji) [reply crypto_guidelines.md]: I pointed out the dialect of the `c` syntax highlighting works.

**Pull request closed without merging**